### PR TITLE
Task/migrate results by item

### PR DESCRIPTION
**Pull request by wtritch**
image

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/results/view/distractor-analysis/distractor-analysis.component.html [hunk 1]
```diff
@@ -39,7 +41,7 @@
           </span>
         </span>
 
-        <p-sortIcon [field]="column.field"></p-sortIcon>
+        <p-sortIcon [field]="column.sortField"></p-sortIcon>
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/assessments/results/view/distractor-analysis/distractor-analysis.component.html]: semi-related.  Noticed a couple of issues in the distractor table impl while working on the results-by-item table.

**Comment by malaffoon** [comment]
Hmm, unrelated but why does it go to 10? Is this in awsqa, awsdev, local dev snapshot?

**Comment by wtritch** [comment]
@malaffoon It's because of the localhost test data:
image

This is in the Grade 11 ELA ICA exam.  Is there a bug here?

**Comment by malaffoon** [comment]
Yeah, i'll add a TODO to fix asmt labels and WER max scores in the dev snapshot.

**Comment by wtritch** [comment]
Merging this now, will resolve comments as they appear.

**Pull request merged**